src/main/ruby/truffleruby/core/truffle/ffi/pointer.rb

```diff
@@ -98,7 +98,11 @@ def inspect
         sign = ''
       end
 
-      "#<#{Primitive.class(self).name} address=#{sign}0x#{addr.to_s(16)}>"
+      if Primitive.pointer_size(self) != UNBOUNDED
+        "#<#{Primitive.class(self).name} address=#{sign}0x#{addr.to_s(16)} size=#{Primitive.pointer_size(self)}>"
```

Comment:
It would be nice to avoid duplication here, like `size = Primitive.pointer_size(self)` and `#{" size=#{size}" if size != UNBOUNDED}`

Reply:
I did it that way first and figured someone would say we don't need to optimize `inspect`, so I went with the shorter approach. Easy enough to change.

Reply:
This wasn't done yet as far as I can see in the diff

Reply:
Fine as-is